docs/modules/ROOT/pages/index.adoc

```diff
@@ -190,15 +192,23 @@ See link:++https://github.com/trustoverip/TechArch/blob/v1-PR1/spec.md#64-toip-i
 
 ==== Credential holder
 
-The _<<_actor,actor>>_ that has control (specifically signature authority) over a _<<ToIP verifiable identifier>>._
+The _<<_actor,actor>>_ that has control (specifically signature authority) over a _<<ToIP verifiable identifier>>_ that describes a specific _<<_named_actor,named actor>>._
 
-==== Credential subject
+==== Identity assertion
 
-The _<<_actor,actor>>_ that is identified by the *subject* field of a _<<ToIP verifiable identifier>>._ This may or may not be the same _<<_actor,actor>>_ as the _<<_credential_holder,credential holder>>._
+A _<<C2PA assertion>>_ that allows a _<<_credential_holder,credential holder>>_ to prove control over an digital identity and bind the identity to a set of _<<_c2pa_assertion,C2PA assertions>>_ produced by them or on their behalf.
 
-==== Identity assertion
+==== Identity assertion consumer
+
+A _<<C2PA Manifest Consumer>>_ who also consumes and interprets the content of any *<<_identity_assertion,identity assertions>>* contained within the _<<C2PA Manifest>>._
```

Comment:
File new TO DO issue: Add discussion about how to distinguish provenance and rights management. What hooks can we provide to rights management platforms?

Reply:
Filed as #108.